docker/README.md

```diff
@@ -0,0 +1,19 @@
+# Build Docker Image
+
+Use the folloiwng steps to run build a GeneFlow docker image:
```

Comment:
Incorrect spelling of "following".  Should "to run build" fragment actually be "to run and build", or "to run a build"?

Reply:
Fixed spelling.